src/main/io/osd.h

```diff
@@ -31,11 +31,12 @@ extern const char * const osdTimerSourceNames[OSD_NUM_TIMER_TYPES];
 #define OSD_PROFILE_COUNT 3
 #define OSD_PROFILE_MAX   ((1 << OSD_PROFILE_COUNT) - 1)
 
-#define VISIBLE_FLAG  0x0800
-#define VISIBLE(x)    (x & VISIBLE_FLAG)
-#define OSD_POS_MAX   0x3FF
-#define OSD_POSCFG_MAX   (VISIBLE_FLAG|0x3FF) // For CLI values
-#define OSD_PROFILE_MASK  (0x07)
+#define VISIBLE_FLAG  0x1
```

Comment:
You should only store the three "profile" bits, these are essentially three separate visibility bits.

Reply:
Updated.